androidgen/pom.xml

```diff
@@ -37,13 +37,6 @@
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
     </dependency>
-    <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.13.2</version>
-      <scope>test</scope>
-    </dependency>
```

Comment:
This is a much needed upgrade from JUnit 4 to JUnit 5. Thanks for making these changes!